tensorrt_llm/_torch/modules/attention.py

```
@@ -656,6 +668,15 @@
         quant_config = config.get_quant_config()
         self.quant_config = quant_config
 
+        # print(f"limin: attention.py, MLA, config.use_cute_dsl_fp8_block_scale_bmm: {config.use_cute_dsl_fp8_block_scale_bmm}, config.use_cute_dsl_fp8_block_scale_gemm: {config.use_cute_dsl_fp8_block_scale_gemm}")
```

Ruff (0.12.2)

Line too long (214 > 120)

(E501)